### ISSUE TITLE
Als ontwikkelaar wil ik weten of alle EIGENSCHAPpen van het ZTC ook in de ZAAK moeten zitten/komen

### ISSUE THREAD

**Issue by joeribekker** [VNG-Realisatie/gemma-zaken#1275]
...zodat ik de standaard goed kan implementeren.

Moet dit of niet? Even goed documenteren en afhankelijk van of dit moet de referentie implementatie aanpassen.

Dit kwam ter spake tijdens overleg over #536 

---
Onderstaande "vink-lijst" is voor het team zelf om in te vullen en hoeft niet door de indiender te worden ingevuld.

**Bepaling prioriteit door PO**

- [ ] verbreding of verdieping API's
- [ ] stimuleert gebruik door gemeenten
- [ ] stimuleert gebruik door leveranciers

... eventueel nog toelichting door PO

**Definition of ready**
- [ ] Iedereen in het team begrijpt de user story
- [ ] de gewenste (aanvulling op de) functionaliteit van de API's duidelijk en beschreven is.
- [ ] Is klein genoeg (maximaal 1/5 van sprint)
- [ ] Product Owner akkoord en voorzien van prioriteit (mag alleen afgevinkt worden door PO)
- [ ] Idee hebben van hoe deze user story kan worden gedemonstreerd.
- [ ] _Globale_ oplossingsrichting bekend
- [ ] Vastgelegd in Github en geplaatst in kolom ready

**Definition of done**
- [ ] Er is een OAS 3.0 specificatie
- [ ] Er is een referentieimplementatie
- [ ] Er zijn tests(cases) aanwezig die de wijziging aantonen en waarmee de user story getest kan worden.
- [ ] De technische specificatie (standaard.md) is gepubliceerd leesbaar
- [ ] Gebruikte gegevensmodel is na **iedere sprint** bijgewerkt.

**Acceptatiecriteria**
- [ ] De DSO URI- en API-strategie worden gevolgd of afwijkingen zijn vastgelegd als ontwerp keuze
- [ ] Er zijn geen bekende GEMMA tegenstrijdigheden **of** afwijkingen zijn vastgelegd.

**Taken**
- [ ] Implementeren in referentie-implementatie [_verantwoordelijke_]
- [ ] Schrijven (unit) test voor referentie-implementatie [_verantwoordelijke_]
- [ ] Genereren/opstellen van OAS 3.0 [_verantwoordelijke_]
- [ ] Human Readable publiceren Open API Specificatie (v.3.0) [_verantwoordelijke_]
- [ ] Documentatie bijwerken
- [ ] Gegevensmodel bijwerken

**Comment by Hugo-ter-Doest** [comment]
Ja, in principe heeft iedere zaak van een bepaald zaaktype de gespecificeerde eigenschappen. Ze hoeven echter geen waarde te hebben. Je kunt bij statustype aangeven welke eigenschappen een waarde moeten hebben voordat je naar die status mag.

**Comment by joeribekker** [comment]
@Hugo-ter-Doest refereert naar https://www.gemmaonline.nl/index.php/Imztc_2.1/doc/relatiesoort/statustype_heeft_verplichte_eigenschap wat op dit moment nog niet in de Catalogi API zit.

**Comment by Hugo-ter-Doest** [comment]
Zaakeigenschappen zijn niet verplicht, maar wel als ze aan een statustype zijn gekoppeld.

**Comment by joeribekker** [comment]
Hugo en ik verwijzen naar 2 verschillende dingen:

1. Antwoord op de vraag "moeten de eigenschappen in het zaaktype altijd overeenkomen met de eigenschappen op de zaak" is het antwoord "ja".
2. Hugo's wil dat we de eigenschappen ook verplicht van waarde voorzien vanaf een bepaalde status

**Comment by Hugo-ter-Doest** [comment]
Deze parkeren we totdat we de discussie over generieke objectregistratie hebben gevoerd.

**Comment by joeribekker** [comment]
Eigenschappen zijn wel echt HEEL makkelijk om te hebben :)

* Bij Den Haag worden ze nu gebruikt voor gegevens die getoond worden aan de initiator (bijv. in een PIP).
* Je kan ze ook alleen gebruiken voor alle gegevens die je wilt bewaren (archief enzo). 

In beide gevallen zijn dit dan wel kopieen van gegevens bij bijv. een Zaak-Object :(

**Comment by EdwinCoster** [comment]
Inderdaad @joeribekker. Ik bemerk ook dat ik dit nu niet correct in het Gegevensmodel heb zitten. Wij hebben ooit bedacht dat er Zaakobjecten zijn voor Inzageverzoek en Mor. maar dat is niet terecht. Inzageverzoek is het Zaaktype, het Zaakobject is een Natuurlijk_Persoon en Eigenschappen zijn (in ieder geval) alle gegevens die je in het e-formulier kunt invoeren die bepalen voor welke domeinen je inzage wil (en in sommige gevallen waarom).

Deze gegevens kun je vervolgens gebruiken om terug te geven aan de aanvrager, en om te bepalen welke domeinen wel en niet onderzocht moeten worden (en dus ook wie er allemaal een vraag krijgen om dat uit te zoeken),

En, inderdaad, mocht je iets moeten bewaren zit alle relevante informatie bij de zaak.

**Comment by michielverhoef** [comment]
> @Hugo-ter-Doest refereert naar https://www.gemmaonline.nl/index.php/Imztc_2.1/doc/relatiesoort/statustype_heeft_verplichte_eigenschap wat op dit moment nog niet in de Catalogi API zit.

In Catalogi API 1.3 is  dit opgenomen.  https://catalogi-api.test.vng.cloud/api/v1/schema/#tag/statustypen/operation/statustype_create 


@MatthijsBekendam Wordt in de Zaken API ook gecontroleerd of bij het bereiken van een Status de verplichte Eigenschappen ook gevuld zijn?
